service/src/main/java/bio/terra/pipelines/service/ImputationService.java

```diff
@@ -61,13 +68,29 @@ public List<ListAppResponse> queryForWorkspaceApps() {
               samService.getTspsServiceAccountToken(),
               imputationConfiguration.workspaceId()));
 
+      // cbas related calls
```

Comment:
just to remind myself, this whole method `queryForWorkspaceApps()` is just a placeholder to make these calls and show they're working, right? eventually these will be moved into flight steps and the uris will be used to make calls etc etc

Reply:
yup yup exactly